### PR TITLE
Fixes #25120 - Horizontal scrollbar appearing on slash command autocomplete

### LINKED ISSUE #25120: Horizontal scrollbar appearing on slash command autocomplete
Environment: Community in Desktop on Mac and Firefox on Mac

Repro steps:

1. Type `/` to open the slash command autocomplete

Observed: There’s a horizontal scrollbar at the bottom of the autocomplete and the descriptions of some slash commands extend off the side of the autocomplete. Depending on the browser, the scrollbar at the bottom may be more or less visible.

Expected: The items in the autocomplete are either truncated with an ellipsis or their description wraps onto another line

----
If you're interested please comment here and come [join our "Contributors" community channel](https://community.mattermost.com/core/channels/tickets) on our daily build server, where you can discuss questions with community members and the Mattermost core team. For technical advice or questions, please  [join our "Developers" community channel](https://community.mattermost.com/core/channels/developers).

New contributors please see our [Developer's Guide](https://developers.mattermost.com/contribute/getting-started/).

JIRA: https://mattermost.atlassian.net/browse/MM-55088

### PR DESCRIPTION
**Pull request by san70sh**
#### Summary
This PR fixes the horizontal scrollbar when the command description overflows while searching for slash commands. Instead, the description is now wrapped.

#### Ticket Link
Fixes https://github.com/mattermost/mattermost/issues/25120
Jira https://mattermost.atlassian.net/browse/MM-55088
-->

#### Screenshots
|  before  |  after  |
|----|----|
| image | image |

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Comment by san70sh** [comment]
I have fixed the linting errors in the scss file, and pushed a new commit to this PR

File: webapp/channels/src/sass/components/_suggestion-list.scss [hunk 1]
```diff
@@ -115,14 +115,17 @@
 }
 
 .slash-command__info {
+    max-width: 89%;
```
Comment (calebroseland) [inline webapp/channels/src/sass/components/_suggestion-list.scss:118]: I'm not sure this `max-width` is necessary—I think `.slash-command__desc`'s `white-space: break-spaces;` should hande things alright

**Review by calebroseland — CHANGES_REQUESTED** [review]

File: webapp/channels/src/sass/components/_suggestion-list.scss (hunk 1 above)
Comment (san70sh) [reply webapp/channels/src/sass/components/_suggestion-list.scss]: Yes, it wasn't necessary. I removed it and pushed a new commit.

**Review by calebroseland — APPROVED** [review]
Thanks, @san70sh!

**Review by crspeller — APPROVED** [review]

**Comment by harshilsharma63** [comment]
@crspeller @calebroseland this PR unintentionally increased the height of each element in Cmd+K channel switcher. Should we revert and revisit this?

**Pull request merged**